src/components/Toast/types.tsx

```diff
@@ -0,0 +1,24 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import { ReactChild } from 'react';
+
+export type ToastProps = {
+    isOpen: boolean;
+    style: ToastStyles;
+    icon: ReactChild;
+    children?: ReactChild;
```

Comment:
```suggestion
    icon: ReactChild;
    children?: ReactNode;
```
I think ReactChild will throw a typeerror if you add more than one child, but maybe that's intended?

Reply:
I agree it should be ReactNode